flake-lang/pre-commit-hooks/rust-monorepo.nix

```diff
@@ -1,7 +1,6 @@
 { inputs, ... }: {
   imports = [
     inputs.pre-commit-hooks.flakeModule # Adds perSystem.pre-commit options
```

Comment:
```suggestion
    inputs.pre-commit-hooks.flakeModule # Adds perSystem.pre-commit options
    ../../settings.nix # might have the path wrong here idk
```

Reply:
oh I just realized that this will mess with the other things in `settings.nix` i.e., it'll start adding `nil`, `shellcheck`, and `purty` in the devTools in the target repo ahah

Reply:
hahaha you're right, lemme see